_dashboards/csp/csp-dynamic-configuration.md

```diff
@@ -1,19 +1,19 @@
 ---
 layout: default
-title: Configuring Content Security Policy rules dynamically
+title: Configuring Content Security Policy directive `frame-ancestors` dynamically
 nav_order: 110
 has_children: false
 ---
 
-# Configuring Content Security Policy rules dynamically
+# Configuring Content Security Policy directive `frame-ancestors` dynamically
 Introduced 2.13
 {: .label .label-purple }
 
-Content Security Policy (CSP) is a security standard intended to prevent cross-site scripting (XSS), `clickjacking`, and other code injection attacks resulting from the execution of malicious content in the trusted webpage context. OpenSearch Dashboards supports configuring CSP rules in the `opensearch_dashboards.yml` file by using the `csp.rules` key. A change in the YAML file requires a server restart, which may interrupt service availability. You can, however, configure the CSP rules dynamically through the `applicationConfig` plugin without restarting the server.
+Content Security Policy (CSP) is a security standard intended to prevent cross-site scripting (XSS), `clickjacking`, and other code injection attacks resulting from the execution of malicious content in the trusted webpage context. OpenSearch Dashboards supports configuring CSP rules in the `opensearch_dashboards.yml` file by using the `csp.rules` key. A change in the YAML file requires a server restart, which may interrupt service availability. You can, however, configure the directive `frame-ancestors` in the CSP rules dynamically through the `applicationConfig` plugin without restarting the server. The support for other directives will be evaluated case by case due to their security implications.
```

Comment:
```suggestion
Content Security Policy (CSP) is a security standard intended to prevent cross-site scripting (XSS), `clickjacking`, and other code injection attacks resulting from the execution of malicious content in the trusted webpage context. OpenSearch Dashboards supports configuring CSP rules in the `opensearch_dashboards.yml` file by using the `csp.rules` key. A change in the YAML file requires a server restart, which may interrupt service availability. You can, however, configure the directive `frame-ancestors` in the CSP rules dynamically through the `applicationConfig` plugin without restarting the server. The support for other directives is evaluated on a case-by-case basis due to security implications.
```